r/configure

```diff
@@ -31,7 +31,7 @@ PKG_DEB_NAME="(unsuppored)"
 PKG_RPM_NAME="(unsuppored)"
 PKG_BREW_NAME="apache-arrow"
 PKG_TEST_HEADER="<arrow/api.h>"
-PKG_LIBS="-larrow -lparquet -larrow_dataset"
+PKG_LIBS="-larrow_dataset -lparquet -larrow"
```

Comment:
If the order matters it worths mentioning in a comment.

Reply:
This was a suggestion from @kou I believe. Happy to include a comment but I'm not confident what it should say.

Reply:
Normally, the order should be resolved automatically by pkg-config, CMake and so on.

To add a comment to this fallback value, `# Must be the same order as $(pkg-config --libs arrow-dataset)` or something will be good.